code/obj/item/mops_cleaners.dm

```diff
@@ -1272,6 +1272,25 @@ TYPEINFO(/obj/item/handheld_vacuum/overcharged)
 		src.set_current_projectile(new /datum/projectile/special/shotchem/wave/wide/emagged)
 		src.projectiles = null
 
+	syndicate
+		name = "\improper GR1-Mk3 Dirt Dispenser"
+		desc = "The Grime Release, Version 1, Mark III Dirt Dispenser, absolute bane of cleanliness.<br>\
+			Must ONLY be used with one of many Syndicate licensed GR1-Mk3 back tanks."
```

Comment:
Are you sure they only work with the correct backtank? as far as I can see theres no special case checking for the syndicate one.

Reply:
No, I mostly referenced the original description. I could put a type check, but I figured what's the harm in letting them mix and match, especially considering the WA-V3 is just worse in terms of blacklisting.